digits/templates/layout.html

```diff
@@ -53,9 +53,21 @@
                 <li><span class="navbar-text">Server: {{server_name}}</span></li>
                 {% endif %}
                 <li><span class="navbar-text">DIGITS version: <br> {{server_version}}</span></li>
-                <li><span class="navbar-text">Caffe version: <br> {{caffe_version}}</span><li>
-                <li><span class="navbar-text">Caffe flavor: <br> {{caffe_flavor}}</span><li>
+                <li><span class="navbar-text">Caffe version: <br> {{caffe_version}}</span></li>
+                <li><span class="navbar-text">Caffe flavor: <br> {{caffe_flavor}}</span></li>
             </ul>
+            </li>
+            <li class="dropdown">
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown">About<span class="caret"></span></a>
+                <ul class="dropdown-menu navbar-inverse">
+                    <li><span class="navbar-text brightness"><a href="https://developer.nvidia.com/digits">
+                        <img src="{{url_for('static', filename='images/nvidia-logo.png')}}"></img>
+                    </a></span></li>
+                    <li><span class="navbar-text"><a href="https://groups.google.com/forum/#!forum/digits-users">DIGITS User Group</a></span></li>
```

Comment:
Can we add `target="_blank"` so they open in a new tab? Opening a new tab when going to a different website feels natural to me.